src/UpgradeExecutor.sol

```diff
@@ -57,4 +57,20 @@ contract UpgradeExecutor is Initializable, AccessControlUpgradeable, ReentrancyG
 
         emit UpgradeExecuted(upgrade, msg.value, upgradeCallData);
     }
+
+    /// @notice Execute an upgrade by directly calling an upgrade contract
+    /// @dev    Only executor can call this.
+    function executeCall(address upgrade, bytes memory upgradeCallData)
```

Comment:
```suggestion
    function executeCall(address target, bytes memory targetCallData)
```

Reply:
Done. Not sure if it'd be better if we also renamed `executeCall` to `targetCall` or something like that